src/state/slices/opportunitiesSlice/constants.ts

```diff
@@ -98,7 +97,7 @@ export const earnLpOpportunity: EarnOpportunityType = {
   cryptoAmountPrecision: '',
   isLoaded: false,
   type: DefiType.LiquidityPool,
-}
+} as LpEarnOpportunityType
```

Comment:
nuh uh, this concerns me - why?

Reply:
Needed as this is really only a subset of `LpEarnOpportunityType` but valid at runtime when we `Object.assign({}, baseOpportunity, actualOpportunity)` to get the missing fields from the original opportunity.
Turns out, there wasn't much we were missing - https://github.com/shapeshift/web/pull/3777